src/templates/package.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
We should drop anything from this file that we cannot determine or adds no value. I think those are: `description`, `homepage`, `license`, `files` and `repository`.  Given the `package.json` lives in the project the user can always add those later. We should probably specify the `version` to be `1.0.0` to align with `npm init` also.